### DISCUSSION TITLE
Labgrid and Microcontrollers

### DISCUSSION THREAD

**Discussion by Bastian-Krause** [labgrid-project/labgrid#1140] [General]
Does labgrid support MCU world well?

_Originally posted by @davidmaze1990 in https://github.com/labgrid-project/labgrid/issues/953#issuecomment-1515770650_

Since this question came up before in the past, I've created this discussion to share common use cases.

**Comment by Bastian-Krause** [comment]
We at @pengutronix come from the Embedded Linux side. You need to decide if labgrid's [resources](https://labgrid.readthedocs.io/en/latest/configuration.html#resources) and [drivers](https://labgrid.readthedocs.io/en/latest/configuration.html#drivers) fit your use cases.

**Reply by jluebbe** [reply]
We've heard of people using labgrid with Zephyr, with some custom drivers.
